analyze_model.py

```diff
@@ -144,3 +144,37 @@ def generate_robustness_winnery_correlation_graph(pruned_robustness,
         pyplot.title("Robustness & Winning Ticket Intersection by Layer", loc='right', fontsize=12, fontweight=0, color='orange')
         pyplot.xlabel("Layer")
         pyplot.savefig(os.path.join(output_dir, filename), format='png')
+
+    @staticmethod
+    def generate_transfer_graph(results: dict, output_dir: str, filename="transfer_fig", base_name="base"):
+        print(results)
+        pyplot.style.use('seaborn-darkgrid')
+        palette = pyplot.get_cmap('Set1')
```

Comment:
Maybe call `pyplot.figure()` here, so subsequent calls to `generate_transfer_graph` don't output the results to the same graph

Reply:
why does it matter where I call it? because of get_cmap? @dorimedini

Reply:
@galshachaf I don't think you actually need to call it if you're just outputting a single plot. But `generate_transfer_graph` may be called more than once in a single experiment, and if that happens then all the results of different calls will be output to the same graph. I had the problem and adding `pyplot.figure()` to the method solved the issue, I could then call my method multiple times and get separate graphs.
If this works as is: no problem. Just test and make sure calling `generate_transfer_graph` twice makes two graphs

Reply:
did you notice it's five lines down in the code?

Reply:
@galshachaf hhhhh whoops